documentation/src/test/java/example/ExternalMethodSourceTestDemo.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2015-2018 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v20.html
+ */
+
+// tag::external_MethodSource_example[]
+package example;
+
+import java.util.stream.Stream;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+
+class ExternalMethodSourceTests {
```

Comment:
Please rename both the _file_ and the _class_ to: `ExternalMethodSourceDemo`.

Example classes in the `documentation` project must be named "*Demo".